python/ec2-tags.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import time
+import sys
+import boto3.ec2
+import requests
+
+
+## Main. ##
+if __name__ == '__main__':
+
+        aws_metadata_endpoint_base_url = "http://169.254.169.254/latest"
+
+        session = boto3.Session()
+        credentials = session.get_credentials()
+
+        credentials = credentials.get_frozen_credentials()
+        access_key = credentials.access_key
+        secret_key = credentials.secret_key
+
+        try:
+            r = requests.put(aws_metadata_endpoint_base_url + '/api/token', headers={'X-aws-ec2-metadata-token-ttl-seconds': '60'})
+            token = r.text
+            r.raise_for_status()
+        except requests.exceptions.HTTPError as err:
```

Comment:
After the expected error, you need to throw an exception for any other errors that may appear. It will look like this:
except Exception as err:
  raise SystemExit(err)
This applies to all exceptions.